spec/helpers/testScheduler-ui.ts

```diff
@@ -104,12 +104,25 @@ module.exports = function(suite) {
       .replace(/\\n/g, '\n');
     }
 
+    function deleteErrorNotificationStack(marble) {
+      const { notification } = marble;
+      if (notification) {
+        const { kind, exception } = notification;
+        if (kind === 'E' && exception instanceof Error) {
+          delete exception.stack;
```

Comment:
This is apparently the offending line of code in some browsers that @kwonoj was talking about. Perhaps for testing purposes we can just copy the object over sans-stack. like... `{ ...exception, stack: null }` or something?